.github/workflows/ember-compare-asset-sizes.yml

```diff
@@ -0,0 +1,32 @@
+name: Ember compare asset sizes
+
+on:
+  workflow_call:
+    secrets:
+      npm-token:
+        required: true
+      ssh-private-key:
+        required: true
+
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
permissions can be defined top level to all workflows in the file (like this) or per job.